usermods/smartnest/usermod_smartnest.h

```diff
@@ -168,4 +172,39 @@ class Smartnest : public Usermod
   {
     return USERMOD_ID_SMARTNEST;
   }
+
+  /**
+   * setup() is called once at startup to initialize the usermod.
+   */
+  void setup() {
+    // Initialization code here
+    if (!initialized) {
+      DEBUG_PRINTF("Smartnest usermod setup initializing...");
+      
+      // Publish initial status
+      sendToBroker("report/status", "Smartnest usermod initialized");
+      
+      initialized = true;
+    }
+  }
+
+  /**
+   * loop() is called continuously to keep the usermod running.
+   */
+  void loop() {
+    // Periodically report status to MQTT broker
+    unsigned long currentMillis = millis();
+    if (currentMillis - lastMqttReport >= mqttReportInterval) {
+      lastMqttReport = currentMillis;
+      
+      // Report current brightness
+      char brightnessMsg[6];
```

Comment:
Max %u output may be 4,294,967,295 which needs 11 characters (including \0)

Reply:
Fixed up